apps/remix-ide/src/app/tabs/runTab/contractDropdown.js

```diff
@@ -300,13 +301,16 @@ class ContractDropdownUI {
       if (error) {
         return this.logCallback(error)
       }
-
+      _paq.push(['trackEvent', 'udapp', 'txTo', this.networkName])
```

Comment:
this will be called when a contract is deployed, but not in the general case where a transaction is sent